apps/prairielearn/src/models/lti13Instance.ts

```diff
@@ -0,0 +1,20 @@
+import { loadSqlEquiv, queryOptionalRow } from '@prairielearn/postgres';
+import { Lti13Instance, Lti13InstanceSchema } from '../lib/db-types';
+
+const sql = loadSqlEquiv(__filename);
+
+export async function selectLti13Instance(lti13_instance_id: string): Promise<Lti13Instance> {
+  const lti13_instance = await queryOptionalRow(
+    sql.select_lti13_instance,
+    {
+      lti13_instance_id,
+    },
+    Lti13InstanceSchema,
+  );
+
+  if (!lti13_instance) {
+    throw new Error(`LTI 1.3 instance ID ${lti13_instance_id} is unavailable`);
+  }
```

Comment:
Any particular reason you're manually checking this instead of using `queryRow`? That function will automatically assert that there is a row and throw an error otherwise.

Reply:
I believe it was so that I could customize the error message. This PR doesn't have an option to 'disable' an instance but that might have been in an earlier design and a cross over for an error that doesn't look like "invalid number of rows"

Reply:
I just realized there's a `src/ee/models` directory so I'll move this there.